src/main/java/Duke.java

```diff
@@ -1,10 +1,67 @@
+import java.util.Scanner;
+import java.util.*;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line;
+        ArrayList<Task> tasks = new ArrayList<>();
+        Scanner in = new Scanner(System.in);
+        final String SPLIT_LINE = "____________________________________________________________";
+        System.out.println(SPLIT_LINE);
+        System.out.println(" Hello! I'm Duke");
+        System.out.println(" What can I do for you?");
+        System.out.println(SPLIT_LINE);
+        line = in.nextLine();
+        while (!line.equals("bye")){
+            if(line.equals("list")){
+                System.out.println(SPLIT_LINE);
+                for (int i = 0; i < tasks.size(); i++) {
+
+                    System.out.println((i+1)+":"+tasks.get(i));
+                }
+                System.out.println(SPLIT_LINE);
+            }else if(line.startsWith("done")){
+                System.out.println(SPLIT_LINE);
+                int taskNumberCompleted = Integer.parseInt(line.replaceAll("\\D+",""))-1;
+                tasks.set(taskNumberCompleted,tasks.get(taskNumberCompleted).completeTask());
+                System.out.println(" Nice! I've marked this task as done:");
+                System.out.println(" "+":"+tasks.get(taskNumberCompleted));
+                System.out.println(SPLIT_LINE);
+            } else if(line.startsWith("todo")){
+                System.out.println(SPLIT_LINE);
+                System.out.println(" Got it. I've added this task:");
+                tasks.add(new Todo(line));
+                System.out.println("  "+new Todo(line));
+                System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
+                System.out.println(SPLIT_LINE);
+//                System.out.println(SPLIT_LINE);
+//                System.out.println("added: "+line);
+//                tasks.add(new Task(line));
+//                System.out.println(SPLIT_LINE);
+            } else if(line.startsWith("deadline")){
+                System.out.println(SPLIT_LINE);
+                System.out.println(" Got it. I've added this task:");
+                tasks.add(new Deadline(line));
+                System.out.println("  "+new Deadline(line));
+                System.out.println(" Now you have " + tasks.size() + " tasks in the list.");
```

Comment:
There is quite a number of repeated lines here, maybe you can refactor into a function.